draft-birkholz-rats-corim.md

```diff
@@ -789,10 +793,629 @@ Not included in the full CDDL definition are CDDL dependencies to CoSWID. The fo
+
+| Range             | Registration Procedures
+|---
+| 0-127    | Standards Action
+| 128-255  | Specification Required
+{: #tbl-iana-corim-entity-map-items-reg-procedures title="CoRIM Entity Map Items Registration Procedures"}
+
+All negative values are reserved for Private Use.
+
+Initial registrations for the "CoRIM Entity Map" registry are provided below.
+Assignments consist of an integer index value, the item name, and a reference to the defining specification.
+
+| Index | Item Name | Specification
+|---
+| 0 | corim.entity-name | RFC-AAAA
+| 1 | corim.reg-id | RFC-AAAA
+| 2 | corim.role | RFC-AAAA
+| 3-255 | Unassigned
+{: #tbl-iana-corim-entity-map-items title="CoRIM Enity Map Items Initial Registrations"}
+
+{: #iana-corim-entity-types-map-items}
+## CoRIM Entity-Types Registry
+
+This document defines a new registry titled "CoRIM Entity Types".
+The registry maintains well-defined integer values as choices for '$entity-name-type-choice' CBOR uints.
+
+Future registrations for this registry are to be made based on {{RFC8126}} as follows:
+
+| Range             | Registration Procedures
+|---
+| 0-127    | Standards Action
+| 128-255  | Specification Required
+{: #tbl-iana-corim-entity-types-reg-procedures title="CoRIM Entity Types Registration Procedures"}
+
+All negative values are reserved for Private Use.
+
+Initial registrations for the "CoRIM Entity Types" registry are provided below.
+Assignments consist of an integer value, the item name, and a reference to the defining specification.
+
+| Index | Item Name | Specification
+|---
+| 0 | corim.manifest-creator | RFC-AAAA
+| 1 | corim.manifest-signer | RFC-AAAA
+| 2-255 | Unassigned
+{: #tbl-iana-corim-entity-types-items title="CoRIM Entity Types Initial Registrations"}
+
+{: #iana-comid-map-items}
+## CoMID Map Items Registry
+
+This document defines a new registry titled "CoMID Map".
+The registry uses integer values as index values for items in 'concise-mid-tag' CBOR maps.
+
+Future registrations for this registry are to be made based on {{RFC8126}} as follows:
+
+| Range             | Registration Procedures
+|---
+| 0-127    | Standards Action
+| 128-255  | Specification Required
+{: #tbl-iana-comid-map-items-reg-procedures title="CoMID Map Items Registration Procedures"}
+
+All negative values are reserved for Private Use.
+
+Initial registrations for the "CoMID Map" registry are provided below.
+Assignments consist of an integer index value, the item name, and a reference to the defining specification.
+
+| Index | Item Name | Specification
+|---
+| 0 | comid.language | RFC-AAAA
+| 1 | comid.tag-identity | RFC-AAAA
+| 2 | comid.entity | RFC-AAAA
+| 3 | comid.linked-tags | RFC-AAAA
+| 4 | comid.triples | RFC-AAAA
+| 5-255 | Unassigned
+{: #tbl-iana-comid-map-items title="CoMID Map Items Initial Registrations"}
+
+{: #iana-comid-xxx-map-items}
+## CoMID Tag-Identity-Map Items Registry
+
+This document defines a new registry titled "CoMID Tag-Identity Map".
```

Comment:
tag-identity-map isn't extensible at the map level but is at the tag-id-type-choice. This is sufficient extensibility.

Reply:
flagged for pruning